Gemfile

```diff
@@ -1,4 +1,5 @@
 source "https://rubygems.org"
 
 gem 'jekyll', '3.8.4'
+gem 'jekyll-redirect-from', '0.14.0'
```

Comment:
How about we use `_redirects` file instead of bringing in this gem?

It's also much nicer to see all redirects in a single place.

Reply:
Good point, switching to that